lib/chat_api/messages.ex

```diff
@@ -6,6 +6,7 @@ defmodule ChatApi.Messages do
   import Ecto.Query, warn: false
 
   alias ChatApi.Repo
+  alias ChatApi.Conversations
```

Comment:
I'm not too sure about aliasing `ChatApi.Conversations` here because that module uses `ChatApi.Messages` too. Feels like there's a circular dependency going on